pkg/controller/job/job_controller.go

```diff
@@ -595,7 +595,7 @@ func (jm *Controller) enqueueSyncJobInternal(logger klog.Logger, obj interface{}
 	// all controllers there will still be some replica instability. One way to handle this is
 	// by querying the store for all controllers that this rc overlaps, as well as all
 	// controllers that overlap this rc, and sorting them.
-	logger.Info("enqueueing job", "key", key, "delay", delay)
+	logger.V(1).Info("enqueueing job", "key", key, "delay", delay)
```

Comment:
As a JobController Reviewer, SGTM on this change since based on logging level documentation, this log seems not to match either level 0 criteria:

```
klog.V(0).InfoS = klog.InfoS - Generally useful for this to always be visible to a cluster operator
    - Programmer errors
    - Logging extra info about a panic
    - CLI argument handling
```

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#what-method-to-use

@mimowo @soltysh @atiratree  Let me know if you have any kind of objections.